crux-core/src/crux/api/ICruxDatasource.java

```diff
@@ -117,7 +120,10 @@
      * @param eid The entity id to return history for
      * @return a cursor of changes to the given entity.
      */
-    public ICursor<Map<Keyword, ?>> openEntityHistory(Object eid, HistoryOptions options);
+    ICursor<Map<Keyword, ?>> openEntityHistory(Object eid, HistoryOptions.SortOrder sortOrder, HistoryOptions options);
```

Comment:
This is a _sneaky_ little breaking change :scream: :wink: 

I think you've managed to keep the whole of the rest of the change non-breaking (nice one!) so let's try avoid it here. I like the idea of the user not needing to create the full HistoryOptions object, but I think we can achieve that with:

```java
ICursor<Map<Keyword, ?>> openEntityHistory(Object eid, HistoryOptions options)
default ICursor<Map<Keyword, ?>> openEntityHistory(Object eid, HistoryOptions.SortOrder sortOrder)
```

Reply:
I was trying to put it in line with the clojure open-entity-history which has the sort order distinct from the rest of the options, but you are right, it *is* breaking.

I'm happy to change it back, though, for consistency with the previous version.

Reply:
Changed it back. :) 

I left in the default versions which just take sort order as they don't break anything and can be convenient rather than having to manually create a `HistoryOption` instance.

Feels a little messy in the Java to Clojure conversion as we effectively take the sort order from the options to pass into `entity-history` as a separate argument only for the implementations to immediately `assoc` it back in ¯\_(ツ)_/¯